### PR TITLE
add ffa example to readme

### PR DESCRIPTION
**Pull request by hanwen-sun**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,13 +107,65 @@ For implementation details, more experimental results and future works, please v
...
+k_ranges=AttnRanges.from_ranges(
+                    [
+                        [0, 128],
+                        [0, 256],
+                        [0, 384],
+                    ]
+                ),
+max_seqlen_q=384
+max_seqlen_k=384
+attn_type_map=torch.tensor([0， 0， 0], device=device)
```
Comment (littsk) [inline README.md]: I feel that we could change the attn_type_map here, add options 0, 1, 2, 3, and then display what kind of mask this is, which would demonstrate the flexibility of the mask.
